### ISSUE TITLE
[feature request] Handle folders in scene files

### ISSUE THREAD

**Issue by scorchio** [kevboh/longform#125]
I've started to import an existing writing project into Longform which has a structure like this:

- part 1 (folder)
  - start.md
  - something happens.md
  - something else happens.md
  - closing part 1.md
- part 2 (folder)
  - another start
  - ...

...you get the idea. 

Not sure if I'm understanding things correctly, but it seems to me that Longform assumes a flat file hierarchy inside `sceneFolder`. No matter how I wanted to add the files to the index file's `scenes` list (I mean with or without the folder name and the slash), it did not work. This would be great to support.

**Comment by pdworkman** [comment]
After you have added the "scenes" to the project or draft, you can drag the child scenes right to indent them and nest them under the parents.

**Comment by kevboh** [comment]
> it seems to me that Longform assumes a flat file hierarchy inside sceneFolder

You are correct. Treating vault subfolders as organizational within a Longform project introduces some complexity that I'm just not prepared to deal with, at least at the current moment. As @pdworkman notes, you can drag scenes or use the indentation commands to adjust them relative to others and simulate folders.

**Comment by b-camphart** [comment]
@kevboh When I finish the big brunt of work I'm focused on now, I might come back to address this.  Would you be happy assigning it to me?

**Comment by kevboh** [comment]
@b-camphart all yours!

**Comment by tustin2121** [comment]
Adding my +1 to this issue. Longform is basically unusable without this feature, especially after trying programs like Scrivener and Manuskript which basically rely on nested folder setups, and thus my projects are already nested.

**Comment by SongshGeo** [comment]
Adding my +1 to this issue. 🙌

**Comment by b-camphart** [comment]
For simplicity, I think longform shouldn't necessarily do anything with the folder structure.  It just scans through for nested files the same way it does now for files within the scene folder.  Probably going to be some headaches around sync, per usual, but I'll know what those are once I get into the code.
